backend/test/models/riff_test.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+# Tests for Riffs
+class RiffTest < ActiveSupport::TestCase
+  #   Assert that a Riff without a created_by (creator) is invalid.
+  test 'a Riff without a created_by (creator) is invalid' do
+    riff = build(:riff)
+    assert_not riff.valid?
+  end
+
+  #   Assert belongs_to :creator (must link to a User via created_by).
+  test 'belongs_to :creator (must link to a User via created_by)' do
+    assoc = Riff.reflect_on_association(:creator)
+    assert_equal :belongs_to, assoc.macro
+    assert_equal 'User', assoc.class_name
+    assert_equal 'created_by', assoc.foreign_key
+  end
+
+  #   Assert belongs_to :latest_revision (optional).
+  test 'belongs_to :latest_revision (optional)' do
+    assoc = Riff.reflect_on_association(:latest_revision)
+    assert_equal :belongs_to, assoc.macro
+    assert_equal 'RiffVersion', assoc.class_name
+    assert_equal 'latest_revision_id', assoc.foreign_key
+  end
+
```

Comment:
I have another idea for a test for latest revision:

test 'allows riff without latest_revision by default' do
  assert build(:riff, latest_revision: nil).valid?
end